src/scripts/create_conll_es.py

```diff
@@ -0,0 +1,105 @@
+"""Create the CoNLL-ES-mini NER dataset and upload it to the HF Hub."""
+
+import pandas as pd
+from datasets import Dataset, DatasetDict, Split, load_dataset
+from huggingface_hub import HfApi
+from requests import HTTPError
+
+
+def main() -> None:
+    """Create the CoNLL-ES-mini NER dataset and uploads it to the HF Hub."""
+    # Define dataset ID
+    repo_id = "conll2003"
```

Comment:
Spanish is part of CoNLL-2002!

```suggestion
    repo_id = "eriktks/conll2002"
```